pkg/db/vulnerability.go

```diff
@@ -24,13 +24,16 @@ func (dbc Config) GetVulnerability(cveID string) (vuln types.Vulnerability, err
 	err = db.View(func(tx *bolt.Tx) error {
 		bucket := tx.Bucket([]byte(vulnerabilityBucket))
 		value := bucket.Get([]byte(cveID))
+		if value == nil {
+			return xerrors.New("no item in the database for this vulnerability")
+		}
 		if err = json.Unmarshal(value, &vuln); err != nil {
-			return xerrors.Errorf("failed to marshal JSON: %w", err)
+			return xerrors.Errorf("failed to unmarshal JSON: %w", err)
 		}
 		return nil
 	})
 	if err != nil {
-		return types.Vulnerability{}, xerrors.Errorf("failed to get the vulnerability: %w", err)
+		return types.Vulnerability{}, xerrors.Errorf("failed to get the vulnerability (%s): %w", cveID, err)
```

Comment:
I'd not use parentheses. something like that:
`return types.Vulnerability{}, xerrors.Errorf("failed to get the information for vulnerability %q: %w", cveID, err)`
but that's just my personal opinion.

Reply:
I'm not very opinionated on this. Quotes sounds fine. Updated.